templates/webapi/test/infopanel.html.ep

```diff
@@ -46,7 +46,10 @@
                     % else {
                         % my $state_detail = '';
                         % if (my @gru_deps = $job->gru_dependencies) {
-                            % $state_detail = join(';', map { 'id: ' . $_->gru_task_id . ', name: ' . $_->gru_task->taskname } @gru_deps);
+                            % my $task = $_->gru_task;
+                            % $state_detail = join(';', map {
+                            %   'id: ' . $_->gru_task_id . ', name: ' . ($task ? $task-taskname : '')
```

Comment:
```suggestion
                            %   'id: ' . $_->gru_task_id . ', name: ' . ($task ? $task->taskname : '')
```

Reply:
Autocompletion in these files is nonexistent, thanks for the catch!